src/app/dashboard/filter-ops/filter-ops.component.html

```diff
@@ -0,0 +1,64 @@
+<div class="filter-ops">
+  <app-load-title title="{{ title }}" [isLoading]="isLoading"></app-load-title>
+
+  <ul class="filter" (clickOutside)="hide()">
+    <li class="dropdown arrow-right">
+      <a (click)="toggle()" [ngClass]="{ active: anyValueSet() }">{{ 'filter' | translate }}</a>
+      <a (click)="clearParams()">{{ 'reset' | translate }}</a>
```

Comment:
There reset button has an dropdown arrow as well. Is that correct?

Reply:
Fixed